tests/test_models.py

```diff
@@ -130,7 +130,7 @@ def test_backoff(self):
                                                     failures=5))
         self.assertFalse(models.should_schedule_next(two_hours_ago, now,
                                                      "3600", failures=10))
-
+        self.assertFalse(models.should_schedule_next(two_hours_ago, now, "3600", failures=32))
```

Comment:
@guyco33 sorry for being a pain in the ass about this, but I'm a firm believer in one assert per test, and specifically, one logical assert per test.

The two assertions above the one you added are kind of related - they test the flip side of the same conditional, and belong in the same test (although this is debatable). The assertion you added feels like it tests a different concept, and I believe it should be separated.

Also - in the absence self-explanatory code, it's the tests' job to provide documentation, and just asserting "false" when failures=32 feels like a magic number with magic behavior. A dedicated test, with an explanatory test name, would really help here.

Reply:
@rauchy I totally agree with you, especially when the tests cover special behaviour of magic numbers and unique corner cases.  
I will pick a proper name for a new assertion and will update the PR accordingly